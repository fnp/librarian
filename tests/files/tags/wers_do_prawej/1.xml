<strofa>Anioł ognisty --- mój anioł lewy/
<wers_do_prawej>Poruszył dawną miłości strunę.</wers_do_prawej>/
Z tobą! o! z tobą, gdzie białe mewy,/
<wers_do_prawej>Z tobą w podśnieżną sybirską trunę,</wers_do_prawej>/
Gdzie wiatry wyją tak jak hyjeny,/
Tam, gdzie ty pasasz na grobach reny.
</strofa>
