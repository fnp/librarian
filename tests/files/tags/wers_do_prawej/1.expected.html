<div id="book-text"><div class="stanza" id="i1">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Anioł ognisty — mój anioł lewy</div>
<div class="wl verse verse-right" id="f2">Poruszył dawną miłości strunę.</div>
<div class="wl verse" id="f3">Z tobą! o! z tobą, gdzie białe mewy,</div>
<div class="wl verse verse-right" id="f4">Z tobą w podśnieżną sybirską trunę,</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Gdzie wiatry wyją tak jak hyjeny,</div>
<div class="wl verse" id="f6">Tam, gdzie ty pasasz na grobach reny.</div>
</div></div>
